remoteappmanager/docker/container_manager.py

```diff
@@ -349,7 +349,7 @@ def _start_container(self,
             '\n'.join('{0} -> {1}'.format(source, target['bind'])
                       for source, target in filtered_volumes.items()))
 
-        container_url_id = _generate_container_url_id()
+        container_url_id = image_name.split("/")[-1]
```

Comment:
I am wondering if there are either any security or practical implications of this change, since the container url ID is no longer a random or unique string.

Are there any best practices for Docker container url ids? I couldn't find much information online. Maybe we can just review any potential information that we are exposing here

Reply:
I agree, however I don't think we're exposing any sensible data, being it simply the name of the image. Also, it shouldn't pose a problem practically (e.g. name conflicts with other containers), since we run only one container (for a given image) at a time. 

I haven't found anything specific about container IDs, so I wouldn't know about best practices, I'm afraid.

Reply:
Yes, that makes sense.

To tidy things up, could we extend the `_generate_container_url_id` to take a `image_name` keyword argument that is `None` by default? If the `image_name` is then provided, then we parse it, if not we just return a random string.